### PR TITLE
fix: Fix a typo in the webchannel-wrapper's package.json.

### PR DESCRIPTION
**Pull request by ehsannas**
Fixes a typo in the webchannel-wrapper's package.json that affected ems5 exports.

### CODE REVIEW

**Review by DellaBitta — CHANGES_REQUESTED** [review]
Please add a changeset with the public release note. Thanks!

**Review by DellaBitta — APPROVED** [review]
Please also add a PR description. Could be the same text as the changeset. Thanks!

**Comment by ehsannas** [comment]
Done.

File: .changeset/orange-rings-own.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@firebase/webchannel-wrapper": patch
+---
+
+fix: Fix a typo in the webchannel-wrapper's package.json that affected ems5 exports.
```
Comment (dconeybe) [inline .changeset/orange-rings-own.md:5]: drive-by comment: there is a typo in the release notes: "ems5" should be "esm5". That is proving to be a difficult acronym to spell correctly.

Comment (dconeybe) [reply .changeset/orange-rings-own.md:5]: @ehsannas

Comment (ehsannas) [reply .changeset/orange-rings-own.md:5]: ouch :(

It seems that this doesn't show up on the release notes (https://firebase.google.com/support/release-notes/js) page though (because it's a webchannel-wrapper package change?)

Comment (dconeybe) [reply .changeset/orange-rings-own.md:5]: Hmm not sure about webchannel-wrapper release notes showing up. Maybe they're not included, but, rather, expect to be mentioned by the dependents upgrading their dependency (i.e. a Firestore change that has its own release note that goes with the webchanne-wrapper upgrade).

Comment (dlarocque) [reply .changeset/orange-rings-own.md:5]: Hi, I wrote the release notes for the 10.12.3 release where this went out. I mistakingly excluded this from the release notes. Looking at older release notes, we list changes to `webchannel-wrapper` under Firestore.

Sorry if I caused any confusion- I'll create a CL to mention this change in the release notes.

Comment (ehsannas) [reply .changeset/orange-rings-own.md:5]: Please add me as reviewer for your CL. Thanks :)

Comment (dlarocque) [reply .changeset/orange-rings-own.md:5]: > Please add me as reviewer for your CL. Thanks :)

Sounds good. I added you as a reviewer
